lzy/lzy-service/src/main/java/ai/lzy/service/BeanFactory.java

```diff
@@ -111,4 +113,10 @@ public OperationDao operationDao(LzyServiceStorage storage) {
     public ObjectMapper mapper() {
         return new ObjectMapper().registerModule(new ProtobufModule());
     }
+
+    @Singleton
+    @Named("PortalChannels")
+    public Map<String, ManagedChannel> portalChannels() {
```

Comment:
I'd suggest to replace generic map with specific type
create, plz, new type `PortalChannels` here